files/default/scripts/run_tests.sh

```diff
@@ -15,7 +15,7 @@ echo "Running tests from $PROXY_TESTS_DIR ..."
 
 test_is_pending() {
   #echo "test_is_pending $1 $2 $3"
-  [[ $1 == "berkshelf" || ( $1 == "chef_client" && $3 == "no_proxy" ) || ( $1 == "install_sh" && $3 == "env_upper") || ( $1 == "chef_client" && $2 == "single" && $3 == "env_upper" ) || ( $1 != "chef_client" && $3 == "client_rb" ) ]]
+  [[ $1 == "kitchen" || ($1 == "berkshelf" && $3 == "no_proxy") || ( $1 == "chef_client" && $3 == "no_proxy" ) || ( $1 == "install_sh" && $3 == "env_upper") || ( $1 == "chef_client" && $2 == "single" && $3 == "env_upper" ) || ( $1 != "chef_client" && $3 == "client_rb" ) ]]
```

Comment:
Not sure. They were failing when we started working on this and we decided that we wouldn't worry about fixing "no proxy" on berks at this time.